apis/r/R/ephemeral.R

```diff
@@ -0,0 +1,450 @@
+#' Ephemeral Collection Base
+#'
+#' @description Base class for ephemeral collections
```

Comment:
please add a few words around what an ephemeral collection _is_: query results; in RAM; not directly associated with soma/seurat/bioc storage formats